### ISSUE TITLE
💡 Improvement: Remove the "default" command

### ISSUE THREAD

**Issue by Madis0** [MarcusOtter/discord-needle#32]
**Is your request for improvement related to a problem? Please describe.**
`/configure default` seems quite dangerous to have as a command, I don't even know if it prompts me to confirm the action. For safety's sake, it could be better to remove it.

**Describe the solution you'd like**
Re-use `/configure message`! 

If the parameter is empty, the bot returns the current value:
![image](https://user-images.githubusercontent.com/8611110/149639977-45beca64-d5d0-4461-842c-fa01329c498f.png)
Well, that message could have the reset button as well.

Even if the user clicked it by accident, they would instantly know _what_ got reset and could recreate it right away.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by MarcusOtter** [comment]
> I don't even know if it prompts me to confirm the action

True, it does not. But it definitely should! Just a simple yes/no confirm dialog should be easy enough to implement and could be very useful for the bot in other commands in the future.

> Re-use `/configure message`!

Actually, the message configuration feature is something I've been meaning to remove. Most users just want to change the language or change how verbose the bot's output is: this can be much easier with something like `/configure language` and `/configure verbosity`. If users want to customize every message individually, forking and self-hosting would be an easier alternative for sure. While I would like to maintain high customizeability I do value simplicty and catering to the "default user". A quick scan through the current configs showed that virtually no server was using customizeable messages (except for auto-threading custom messages on a per-channel basis, which I intend to keep)

>  Well, that message could have the reset button as well.

This is a neat idea as well. A bit trickier to implement so the confirmation dialog should be enough. If someone *does* accidentally delete their configuration, I do take atomatic and regular backups on the VPS, so one could pop into the Discord server and get it restored fairly easily 👍

**Comment by MarcusOtter** [comment]
To put it succinctly: rather than removing the `default` command I think we should add a confirm/cancel prompt to it with buttons. We can also add information about how to restore it (join the Discord and contact me). If it becomes a common problem, we can look into it more.

**Comment by Madis0** [comment]
> If users want to customize every message individually, forking and self-hosting would be an easier alternative for sure. While I would like to maintain high customizeability I do value simplicty and catering to the "default user". A quick scan through the current configs showed that virtually no server was using customizeable messages (except for auto-threading custom messages on a per-channel basis, which I intend to keep)

I agree, it was confusing to see error messages being customizable, but I think the success messages could still stay customizable. For example, I consider the default archive success message way too verbose and I shortened it down to one sentence.

**Comment by MarcusOtter** [comment]
I'd like `/configure verbosity` to have something like a one-liner setting so you only get the essential stuff like you mentioned, which would be applied globally. It could even have a `No visible messages`-setting that would only create hidden messages when replying to commands and nothing else. Sounds like that would suit your use case as well.

**Comment by ririko5834** [comment]
You can use 2 buttons with ✅ and ❌ emojis for confirmation prompts.
